wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -34,6 +34,8 @@ library
                        Cardano.Wallet.API.V1.Errors
                        Cardano.Wallet.API.V1.Updates
                        Cardano.Wallet.API.V1.Migration
+                       Cardano.Wallet.API.V1.Migration.Helpers
```

Comment:
@akegalj I don't like "helper" modules, because they feel largely artificial . Let's call it simply "Cardano.Wallet.API.V1.Migration.Monads", as it would convey the idea that we are migrating between different monadic stacks 😉